frontend/tet/admin/.eslintrc.js

```diff
@@ -0,0 +1,57 @@
+const { join } = require('path');
+
+module.exports = {
+  extends: [
+    'auto',
+    'plugin:@next/next/recommended',
+    'plugin:you-dont-need-lodash-underscore/compatible',
+  ],
+  rules: {
+    'jsx-a11y/anchor-is-valid': [
+      'error',
+      {
+        components: ['Link'],
+        specialLink: ['hrefLeft', 'hrefRight'],
+        aspects: ['invalidHref', 'preferButton'],
+      },
+    ],
+    'import/no-extraneous-dependencies': [
+      'error',
+      // Use package.json from both this package folder and root.
+      { packageDir: [__dirname, join(__dirname, '../../')] },
+    ],
+    'no-void': 'off',
```

Comment:
there were some rules recently added to the other projects as new major version of eslint was taken into use.

Reply:
Thanks, I will review the rules under tet apps!